package.json

```diff
@@ -3,11 +3,17 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@reduxjs/toolkit": "^1.9.5",
```

Comment:
please test if this one is needed

Reply:
the use of the toolkit is recommended in newer versions of redux

Reply:
What does it do?

Reply:
And can we not move it to `devDependencies`?

Reply:
don't need it at the moment